scss/_debug.scss

```diff
@@ -0,0 +1,44 @@
+@if $enable-debugger-classes {
+  .debug {
```

Comment:
more specific naming like `.bs-debug` or `.grid-breakpoint-debug` will be good to avoid conflicts of user space class names.

Reply:
Yes definitely, i'm changing it.